src/pages/ManageApps.vue

```diff
@@ -135,15 +130,19 @@ appsStore.$subscribe(() => {
                 <VSeperator vertical class="stats-separator" />
                 <VStack direction="column" align="center">
                   <span class="stats-title">Estimated Cost</span>
-                  <span class="stats-number">$0</span>
+                  <span class="stats-number"
+                    >${{ app.estimatedCost || 0 }}</span
```

Comment:
Is `estimatedCost` missing in any case from the api response?

Reply:
Removed here https://github.com/arcana-network/developer-dashboard/pull/135/commits/134e776175d244327e99ab2630bdd290597ff539

Reply:
@ajithranka no, I just confirmed, estimated_cost won't be missing from api response, it'll always be there

Reply:
We can remove `|| 0` in that case.